src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/VolleyUnitOfWork.cs

```diff
@@ -54,7 +54,14 @@ public Task CommitAsync()
         /// </summary>
         public void Dispose()
         {
-            _context.Dispose();
+            Dispose(true);
+        }
+        protected virtual void Dispose(bool disposing)
```

Comment:
Take a look to dispose pattern.

Reply:
+1

who calls this method with false parameter?

Reply:
ok, reverting